framework/src/postprocessors/ElementW1pError.C

```diff
@@ -34,7 +34,8 @@ ElementW1pError::ElementW1pError(const InputParameters & parameters) :
 Real
 ElementW1pError::getValue()
 {
-  return std::pow(ElementIntegralPostprocessor::getValue(), 1./_p);
```

Comment:
The right way would be to make then save the return value of that call to a local variable, then apply the scaling factor as you are doing before you return. The call should still be in there.